src/rightClick.js

```diff
@@ -1,8 +1,23 @@
 // Register the Plugin when installed
 chrome.runtime.onInstalled.addListener(() => {
-	// TODO: GET INSTANCE ID FROM LAMBDA
-	chrome.storage.local.set({ instanceID: "xxxxxxxx-xxxx-Mxxx-Nxxx-xxxxxxxxxxxx" });
-	mapMenus();
+	fetch("https://439r656kxf.execute-api.us-east-2.amazonaws.com/dev/device",
```

Comment:
Food for thought.  In a future version, let's move to a function that determines the host.  That way we only need to change the URL in one place.